src/qibo/validation/gate_set_tomography.py

```diff
@@ -0,0 +1,229 @@
+from itertools import product
+
+import numpy as np
+
+from qibo import gates, symbols
+from qibo.backends import GlobalBackend
+from qibo.hamiltonians import SymbolicHamiltonian
```

Comment:
I advise avoiding the use of symbolic Hamiltonians as much as possible, because they're really slow